ros2_control_demo_robot/package.xml

```diff
@@ -14,6 +14,7 @@
   <depend>hardware_interface</depend>
   <depend>pluginlib</depend>
   <depend>rclcpp</depend>
+  <depend>gazebo_ros2_control</depend>
```

Comment:
you're missing an `<exec_depend>xacro</exec_depend>` here.

Reply:
Thanks. Fixed in e6497f1e570cd3e4e2afeb1d4deeaaf6161c1bf3